Feature/164 new tip event

ahoseinian opened the pull request:
Listen to tips.newTip and send push event on litentry_test network.

ahoseinian committed:
Listen to tips.newTip event and send push notification to respective topic.

fixes https://github.com/litentry/litentry-app/issues/164

ahoseinian committed:
Update config

fixes https://github.com/litentry/litentry-app/issues/164

silva-fj commented on src/chain/config.ts:
```diff
@@ -33,6 +33,18 @@ const litentryNetwork: ChainConfig = {
+    getPushData() {
+      return {
+        topic: "tips.newTip",
+        data: { deeplink: "litentry://api/litentry_test/tips" },
```
All these events will have the same title and body for all networks. This payload should be constructed in a way that can be reused for all networks, the network should come from the env variable (if we decide to keep that approach). Later we can come up with a better format for the links

ahoseinian replied:
I have changed this on 188e76c please check if it's better now.

ahoseinian committed:
Update eslint config for new eslint-config-prettier

more info on : https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21

ahoseinian committed:
Move push events to their own file and address PR issue

fixes https://github.com/litentry/litentry-app/issues/164

geastwood reviewed (approved):
LGTM

silva-fj commented on src/chain/pushEvents.ts:
```diff
@@ -0,0 +1,40 @@
+  return [treasuryProposedEvent(networkKey), tipsNewTipEvent(networkKey)];
+};
+
+function tipsNewTipEvent(networkKey: string) {
```
The event configs can be just plain objects, currently you can get the network from `process.env.CHAIN_NETWORK`

ahoseinian replied:
7d54354 
done ✅

ahoseinian committed:
Read push network from .env

litentry/litentry-app#164
